### PR TITLE
add simple-hint@3.0.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by rajpatel507**
Pull request for issue: #10204
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/catc/simple-hint/
 * Official website (optional, not the repository):https://catc.github.io/simple-hint/
 * NPM package url (optional):
 * License and its reference:https://github.com/catc/simple-hint/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of stars:145
   - Count of watchers:12
   - Count of forks:5
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/simple-hint/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "simple-hint",
+    "filename": "simple-hint.min.css",
+    "version": "3.0.0",
+    "private": true,
```
Comment (pvnr0082t) [inline ajax/libs/simple-hint/package.json]: Let's remove `"private"` field because we don't need this. Thanks.

**Comment by rajpatel507** [comment]
@pvnr0082t  requested changes are committed, please review.

File: ajax/libs/simple-hint/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ],
+    "autoupdate": {
+        "source": "git",
+        "target": "git://github.com/catc/simple-hint.git",
+        "fileMap": [
+          {
+            "basePath": "dist",
+            "files": [
+              "simple-hint.min.css",
+              "simple-hint.css"
```
Comment (pvnr0082t) [inline ajax/libs/simple-hint/package.json]: Let's use `"**/*"` instead of these two. It'll be more flexible.

**Comment by rajpatel507** [comment]
Done with changes please review.

File: ajax/libs/simple-hint/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name":"simple-hint",
```
Comment (pvnr0082t) [inline ajax/libs/simple-hint/package.json]: @rajpatel507 There should be a space after colon mark. Please fix all the fields. Thanks.

**Comment by rajpatel507** [comment]
@pvnr0082t Changed package.json, please review.

File: ajax/libs/simple-hint/package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  "description": "CSS-only tooltip packed with a variety of features.",
+  "version": "3.0.0",
+  "homepage": "https://catc.github.io/simple-hint/",
+  "author": {
+    "name": "Catalin Covic",
+    "url": "https://github.com/catc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/catc/simple-hint"
```
Comment (kennynaoh) [inline ajax/libs/simple-hint/package.json]: I think `https://github.com/catc/simple-hint.git` is better.

**Comment by pvnr0082t** [comment]
@rajpatel507 The CI test still failed. Have you tried to run `./tools/fixFormat.js`? If there is any problem you met, feel free to ask.
BTW, if you do some changes, please leave some message or ping us to reivew. Thanks.

**Comment by rajpatel507** [comment]
@pvnr0082t I have noticed that CI still failing so I am cloning my fork on my local machine to run `./tools/fixFormat.js` , previously i was doing changes to Github GUI.

Now when I run `node ./tools/fixFormat.js` its giving some error ,please tell me right way to run `./tools/fixFormat.js` Note that i have sparseCheckout will it be affecting scripts?

**Comment by PeterDaveHello** [comment]
@rajpatel507 What's the error message?

**Comment by rajpatel507** [comment]
@pvnr0082t @PeterDaveHello I have updated package.json after running `./tools/fixFormat.js`. Please Review.

**Comment by PeterDaveHello** [comment]
@pvnr0082t can you help @rajpatel507 squash and rebase the branch? Thanks.

**Comment by PeterDaveHello** [comment]
@kennynaoh @x09326 ?

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@pvnr0082t @kennynaoh did you confirm the hash of the file with upstream?

**Comment by pvnr0082t** [comment]
Yes, I have confirmed and they are the same.

**Comment by PeterDaveHello** [comment]
@pvnr0082t Show me your result ?

**Comment by pvnr0082t** [comment]
Oops, I haven't checked `simple-hint.min.css`. They are not the same...
```
md5sum simple-hint.*
c41d5d1d9bdc618d3a2176774090697f  simple-hint.css
c41d5d1d9bdc618d3a2176774090697f  simple-hint.css_origin
c476d25564c295ded71cda52fd7f1f92  simple-hint.min.css
e42f82f1537659879dbbf791a32a4f12  simple-hint.min.css_origin
```
```
sha1sum simple-hint.*
2883bcba618e1971caf6defe1317c1850c18e8a9  simple-hint.css
2883bcba618e1971caf6defe1317c1850c18e8a9  simple-hint.css_origin
5ac2aba06bd8d0e64a2b1c7cdce73cdc9b8a888a  simple-hint.min.css
72ba2a4391a7857341d88041f97718b76478cfa0  simple-hint.min.css_origin
```

**Comment by PeterDaveHello** [comment]
@pvnr0082t They don't even have the same size, should be more careful ...

**Comment by PeterDaveHello** [comment]
@rajpatel507 would you please fix that problem we mentioned?

**Comment by rajpatel507** [comment]
@pvnr0082t  I have got `simple-hint.min.css` from same repo but using crome 'save as' to download file.
now I have cloned repo to my local and updated file from them please check now.

**Comment by PeterDaveHello** [comment]
Congratulations @rajpatel507 ! Thanks for your contribution :tada: !

**Pull request merged**